components/tag/src/stories.tsx

```diff
@@ -13,11 +14,109 @@ export default {
 export const Default: Story = (args) => <Tag {...args} />;
 Default.args = {
   children: 'beta',
-  inactive: false,
 };
 
 export const Inactive: Story = (args) => <Tag {...args} />;
 Inactive.args = {
-  children: 'beta',
-  inactive: true,
+  children: 'inactive',
+  tint: 'GREY',
```

Comment:
by deprecating `inactive` we are implying it will still work for now but will be removed in a future update.

Please can we keep the visual regression test for `inactive: true`. If you want, you could add a note to the story that this is deprecated:

```ts
Inactive.parameters = {
  docs: {
    storyDescription: 'deprecated',
  },
};
```